### DISCUSSION TITLE
Statistics: Suggestion to remove ambiguity of data

### DISCUSSION THREAD

**Discussion by lamyergeier** [FreshRSS/FreshRSS#6902] [Ideas]
# Reference Image

![image](https://github.com/user-attachments/assets/01c2bc60-405b-4b4f-86c6-e44831e43281)

# Idea

In the above screenshot, Top 10 Feeds shows all time data from the beginning when we started to use Freshrss. So, if some feeds were added later there data won't show up even if they are having a lot of entries per feed.

To avoid this ambiguity, may be provide a way to enter a date from which the data should be computed (by default it could show the all time data.

# Purpose

This can be useful to identify, which feed has most entries in the last month, and thus user can take relevant action.

**Comment by math-GH** [comment]
let's discuss it as an issue here: https://github.com/FreshRSS/FreshRSS/issues/6906

**Comment by benjuade** [comment]
Furthermore, when one imports a feed a certain number of articles are unread by default. Thus, if I marked them as read in order to start with a clean new subscription, those article counts against the total. This is meaningless..
